DS/AdvancedDataStructures/ParserForTruthTable/Validation.java

```diff
@@ -0,0 +1,69 @@
+public class Validation {
+
+	/** Method to validate expression
+	 * 
+	 * @param expression : expression to validate 
+	 * @return true if validated, false otherwise
+	 */
+	public static boolean isExpression(String expression) 
+	{
+		if(expression == null)
+			return false;
+		char[] expressionArray = expression.toCharArray();
+		int length = expressionArray.length;
+	    int openCount = 0, closeCount = 0;
+	    int i = 0;
+	    
+	    /* validation for two consecutive letters or numbers */
+	    for(i = 0; i < (length-1); i++)
+	    {
+			
+	    	if((Character.isLetter(expressionArray[i])))
+	    	{
+	    		if(Character.isLetter(expressionArray[i+1]) || expressionArray[i+1] == '(')
+	    				return false;
+	    	}
+	    	if(expressionArray[i] == ')')
```

Comment:
can we combine it with the other if(expressionArray[i] == ')' || Character.isLetter(expressionArray[i]))